src/vs/workbench/parts/snippets/electron-browser/configureSnippets.ts

```diff
@@ -221,13 +225,15 @@ CommandsRegistry.registerCommand(id, async (accessor): Promise<any> => {
 		existing.push({ type: 'separator', label: nls.localize('new.global.sep', "New Snippets") });
 	}
 
-	const pick = await quickInputService.pick(([] as QuickPickInput[]).concat(existing, globalSnippetPicks, picks.future), {
+	const pick = await quickInputService.pick(([] as QuickPickInput[]).concat(existing, globalSnippetPicks, workspaceSnippetPicks, picks.future), {
 		placeHolder: nls.localize('openSnippet.pickLanguage', "Select Snippets File or Create Snippets"),
 		matchOnDescription: true
 	});
 
 	if (globalSnippetPicks.indexOf(pick as GlobalSnippetPick) >= 0) {
-		return createGlobalSnippetFile((pick as GlobalSnippetPick).uri, windowService, notificationService, fileService, opener);
+		return createSnippetFile('global', (pick as GlobalSnippetPick).uri, windowService, notificationService, fileService, opener);
+	} else if (workspaceSnippetPicks.indexOf(pick as WorkspaceSnippetPick) >= 0) {
+		return createSnippetFile((pick as WorkspaceSnippetPick).name + ' workspace', (pick as WorkspaceSnippetPick).uri, windowService, notificationService, fileService, opener);
```

Comment:
also localization, maybe better with the ` + workspace` part because it's hard to tell how this will look in various languages or with different workspace names